plugins/aea-ledger-solana/setup.py

```diff
@@ -37,9 +37,9 @@
         "open-aea>=1.0.0, <2.0.0",
         "cryptography",
         "PyNaCl==1.5.0",
-        "solana==0.28.1",
-        "solders==0.10.0",
-        "anchorpy==0.14.0",
+        "solders>=0.14.0",
+        "solana>=0.29.0",
```

Comment:
This still have a dependency conflict with the `apduboy` package because of `construct` and `construct-typing` packages but it's won't be an issue for the agent runtime since the ledger wallet plugin is CLI only